SUPPORT.md

```diff
@@ -4,63 +4,63 @@
+2. **Active** - The SDK/CLIs are generally available and fully supported, will receive new feature updates, as well as bug and security fixes. The major version will remain active for at least 12 months from the release date. Compatible updates for the major release are provided through minor versions, or patch versions. Customers are encouraged to use the latest version as that is the version that will get fixes and updates.
+3. **Deprecated** - A library has been superseded by a more recent release. In this case, the current library is deprecated in favor of a newer library. Typically, deprecation occurs at the same time the replacement library is transitioned to Active, after which the releases will only address the most critical bug fixes and security fixes for at least another 12 months.
+4. **Community** – SDK/CLI will no longer receive updates from Microsoft unless otherwise specified in the separate customer agreement. The package will remain available via public package managers and the GitHub repo, which can be maintained by the community.
 
-Teams Toolkit CLI and SDK depend on Microsoft Teams Platform, Teams Dev Portal, SharePoint Framework, Microsoft Azure, programming language runtime, OS, and third-party libraries. 
+## CLI/SDK dependencies
 
-Teams Toolkit CLI/SDK will not be guaranteed to work on platforms and other dependencies that have reached their end of life. Dropping support for such dependencies may be done without increasing the major version of CLI/SDK. We strongly recommend migration to supported platforms and other dependencies to be eligible for technical support.
+Teams Toolkit CLI and SDK depend on Microsoft Teams Platform, Teams Dev Portal, SharePoint Framework, Microsoft Azure, programming language runtime, OS, and third-party libraries.
 
+Teams Toolkit CLI/SDK will not be guaranteed to work on platforms and other dependencies that have reached their end of life. Dropping support for such dependencies may be done without increasing the major version of CLI/SDK. We strongly recommend migration to supported platforms and other dependencies to be eligible for technical support.
 
 *support means we apply critical patches, fix critical bugs, but do not add new features.*
 
 ## Version mapping
 
 | | Teams Toolkit for Visual Studio Code|Teams Toolkit for Visual Studio| Teams Toolkit CLI | TeamsFx SDK |Teams SDK|Manifest|
 |----|----|----|----|----|----|----|
-|Public Preview|v3.8.x|v17.2|v0.14.x|v0.7.x|v1.11.x,v1.12.x|v1.11|
+|Public Preview|v3.8.x|v17.2|v0.14.x|v0.7.x|v1.11.x, v1.12.x|v1.11|
 |GA|v4.0.0|v17.3|v1.0.0|v1.0.0|v1.12.x|v1.11|
-|Beta*|v4.0.0|v17.3|v1.0.0|v2.0.0-beta|v2.0.0|m365DevPreview|
+|Latest*|v5.x.x|v17.6|v2.x.x|v2.x.x|v2.x.x|v1.16|
+|Beta**|Prerelease|v17.7 Preview x|v2.x.x-beta|v2.x.x-beta|v2.x.x-beta|devPreview|

```

Comment:
nit: it's a little bit confused for me about the `x` in `v17.7 Preview x`, how about using `v17.7 Preview` directly?

Reply:
Sounds good, updated.